next.config.js

```diff
@@ -1,11 +1,16 @@
 const checkEnvVariables = require("./check-env-variables")
+const nextIntl = require("next-intl/plugin")
+
 
 checkEnvVariables()
 
 /**
  * @type {import('next').NextConfig}
  */
-const nextConfig = {
+
+const withNextIntl = nextIntl("./src/lib/i18n/request-config.js")
```

Comment:
Why not go with the default `./src/i18n/request.ts`? By following the default path it might make it easier to align with what's commonly done in projects.